### PR TITLE
Introduction and Data Analaysis

### LINKED ISSUE #5: Introduction and data reading
+ introduction
+ data section
+ data pre-processing

(See more details in #8)

### PR DESCRIPTION
**Pull request by diegocasmo**
Closes #5 

+ introduction
  - [x] what problem are we trying to solve?
  - [x] motivate why is it interesting
  - [x] state contributions (later on)
  - [x] Use citation when referring by author name

+ data section
  - [ ] description of the problem in more detail
  - [ ] discuss data attributes (features, labels, etc)

+ data pre-processing
  - [x] Separate images into training, validation, and test sets
  - [x] Example code of how to load, crop, and down-sample an image

### CODE REVIEW

File: report/sections/data.tex [hunk 1]
```diff
@@ -1,9 +1,23 @@
...
+
+This means that we’re looking at a regression problem, not a classification problem: we don’t have to determine which classes the galaxies belong to, but rather the fraction of people who would classify them as such.
+
+\subsection{Preprocessing}
+
+In each image, the object of interest is centered. We can exploit this characteristic by subsampling the image essentially carving off unecessary information in each image which could impair our network in.
+
+image down-sampling (non-destructive cropping to reduce the number of parameters feed to the network)
+
+Down sampling helps the CNN learn which regions are related to each specific expression and also enables the CNN to be performed on the GPU more efficiently \cite{deep-learning-review}.
```
Comment (tristaaan) [inline report/sections/data.tex:19]: Is this the right project?

File: report/references.bib [hunk 2]
```diff
@@ -68,56 +68,3 @@ @misc{zooniverse
   howpublished={\url{https://www.zooniverse.org/projects/zookeeper/galaxy-zoo/about/research}},
   notes={Accessed April 25, 2018.}
 }
-
-@ARTICLE{sdss-segue-1,
```
Comment (tristaaan) [inline report/references.bib]: removed this insane citation, I was only citing the year anyway.

File: report/sections/data.tex (hunk 1 above)
Comment (diegocasmo) [reply report/sections/data.tex:19]: I'm not sure, I do not recall reading that paper (I get a warning due to an undefined reference, I suppose you do too?)

Comment (tristaaan) [reply report/sections/data.tex:19]: Whoops, I made a bibtex entry but did not save it to the file, it's added now. It replaces the url that was there. I wanted to announce when I did this too that urls should not be pasted in the paper but pushed into the .bib file or the references.md file. (cc @ankurshukla03)

Comment (ankurshukla03) [reply report/sections/data.tex:19]: Ok i will add the urls or any article in .bib file from now on.

**Pull request closed without merging**